packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -96,6 +88,27 @@ export abstract class DeployRetrieveExecutor<
     }
   }
 
+  async setApiVersionOn(components: ComponentSet) {
+    // Check the SFDX configuration to see if there is an overridden api version.
+    // Project level local sfdx-config takes precedence over global sfdx-config at system level.
+    const userConfiguredApiVersion = await this.getUserConfiguredApiVersion();
+    
+    // If no user-configured Api Version is present, then get the version from the Org.
+    const orgApiVersion = await this.getOrgApiVersion();
+    components.apiVersion = userConfiguredApiVersion ?? orgApiVersion;
+  }
+
+  async getUserConfiguredApiVersion() {
+    return await ConfigUtil.getConfigValue('apiVersion') as
```

Comment:
Beggars can't be choosers, but could this move to one of the ConfigUtil's as a specifically named method?

Reply:
Yep, good call!  Moved it to ConfigUtil